doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_dfu.rst

```diff
@@ -10,7 +10,7 @@ Device Firmware Update using SUIT
 The Software Updates for Internet of Things (SUIT) is the Device Firmware Upgrade (DFU) and secure boot method for nRF54H Series of System-on-Chip.
 The SUIT procedure features a script-based system that resides in a :ref:`SUIT manifest <ug_nrf54h20_suit_manifest_overview>`.
 The manifest contains instructions for the DFU and booting procedure, and can be customized.
-It includes the use of :ref:`sequences <ug_suit_dfu_suit_concepts_sequences>` (conditions and directives) as well as the concept of :ref:`components <ug_suit_dfu_component_def>`.
+It includes the use of :ref:`sequences <ug_suit_dfu_suit_concepts_sequences>` (built of conditions and directives) as well as the concept of :ref:`components <ug_suit_dfu_component_def>`.
```

Comment:
```suggestion
It includes the use of :ref:`sequences <ug_suit_dfu_suit_concepts_sequences>` (built by the conditions and directives) as well as the concept of :ref:`components <ug_suit_dfu_component_def>`.
```

Reply:
I don't think this is the meaning I wanted to achieve. Maybe built from?

Reply:
Actually, it is fine as is